File: api/code-samples/python/eth/account/getBalance.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = "0x1cbd3b2770909d4e10f157cabc84c7264073c9ec"
blockNumberOrHash = "latest"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_balance(address, blockNumberOrHash)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/account/getBalance.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const address = '0x1cbd3b2770909d4e10f157cabc84c7264073c9ec'
    const blockNumberOrHash = 'latest'

    sdk.klay.getBalance(address, blockNumberOrHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()